net/crowdsec/files/crowdsec.defaults

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+CONFIG=/etc/crowdsec/config.yaml
+data_dir=`uci get "crowdsec.crowdsec.data_dir"`
+sed -i "s,^\(\s*data_dir\s*:\s*\).*\$,\1$data_dir," $CONFIG
+db_path=`uci get "crowdsec.crowdsec.db_path"`
+sed -i "s,^\(\s*db_path\s*:\s*\).*\$,\1$db_path," $CONFIG
+
+# Create data dir & permissions if needed
+if [ ! -d "${data_dir}" ]; then
+	mkdir -m 0755 -p "${data_dir}"
+fi;
+
+grep -q "login:" /etc/crowdsec/local_api_credentials.yaml
+if [ $? -eq 0 ]; then
+	echo local API already registered...
+else
+	if [ -f /var/lib/dbus/machine-id ]; then
+		cscli -c /etc/crowdsec/config.yaml machines add "$(cat /var/lib/dbus/machine-id)" -a -f /etc/crowdsec/local_api_credentials.yaml
+	else
+		cscli -c /etc/crowdsec/config.yaml machines add -a -f /etc/crowdsec/local_api_credentials.yaml
+	fi
+fi
+if [ -s /etc/crowdsec/online_api_credentials.yaml ]; then
+	echo online API already registered...
+else
+	cscli -c /etc/crowdsec/config.yaml capi register -f /etc/crowdsec/online_api_credentials.yaml
+fi
+cscli hub update && cscli collections install crowdsecurity/linux && cscli parsers install crowdsecurity/whitelists && cscli hub upgrade
```

Comment:
@jefferyto 
I mainly modified only this, to handle a little better the first installation and later installation (upgrade)...
The local and online API need to be only registered once !

Reply:
Sounds good to me

Reply:
thanks.